### DISCUSSION TITLE
testing the webhook

### DISCUSSION THREAD

**Discussion by Aemiii91** [OnionUI/Onion#1182] [Support]
### Checklist

- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and something is incorrect or missing

### Wiki Section

no

### Description

no

### Correction

no

**Comment by thomasklemenc** [comment]
did you try turning it off and on again?

**Comment by schmurtzm** [comment]
Each issue message triggers a notification ?
